subprojects/provider/src/test/kotlin/org/gradle/kotlin/dsl/integration/ScriptCachingIntegrationTest.kt

```diff
@@ -38,6 +39,7 @@ class ScriptCachingIntegrationTest : AbstractIntegrationTest() {
         val isolatedTestKitDir = IsolatedTestKitDir()
     }
 
+    @Ignore("partial-evaluator-wip")
```

Comment:
I'll have a look at fixing those tests, a good way to get to know the partial evaluator.